### ISSUE TITLE
Support granting permissions on schema

### ISSUE THREAD

**Issue by paul-pop** [cyrilgdn/terraform-provider-postgresql#12]
### Terraform Version
Terraform v0.13.5

### Affected Resource(s)
- postgresql_grant

### Terraform Configuration Files
This should be as simple as adding a new object type called `schema` so it should look like this:

```hcl
resource "postgresql_grant" "grant" {
  role        = "test"
  database    = "test"
  schema      = "public"
  object_type = "schema"
  privileges  = ["USAGE", "CREATE"]
}
```

Which should execute this under the hood:

```sql
GRANT USAGE,CREATE ON SCHEMA public TO test;
```

See https://www.postgresql.org/docs/9.1/sql-grant.html for supported priviledges.

**Comment by cyrilgdn** [comment]
@paul-pop Thanks for opening this issue.

I'll work on this as soon as possible, it's also required by #16 

However, it's already possible to manage privileges on schema with the [`policy` block in the `postgresql_schema` resource](https://registry.terraform.io/providers/cyrilgdn/postgresql/latest/docs/resources/postgresql_schema).

But I'd prefer to have this possibility in `postgresql_grant` too, and it's a bit buggy for an existing schema (e.g. the `public` one in your case). My goal is to remove this policy block.

**Comment by paul-pop** [comment]
Thank you @cyrilgdn! That’s how we have managed to do it now, using the policy block.

**Comment by cyrilgdn** [comment]
@paul-pop

FYI, this has just been released in [v1.11.0](https://registry.terraform.io/providers/cyrilgdn/postgresql/latest).
